template.js

```diff
@@ -229,12 +229,12 @@
 
     // Taken from https://github.com/jquery/jquery/blob/73d7e6259c63ac45f42c6593da8c2796c6ce9281/src/manipulation/wrapMap.js
     var topLevelWrappingMap = {
-      ['option']: ['select'],
-      ['thead']: ['table'],
-      ['col']: ['colgroup', 'table'],
-      ['tr']: ['tbody', 'table'],
-      ['th']: ['tr', 'tbody', 'table'],
-      ['td']: ['tr', 'tbody', 'table']
+      'option': ['select'],
```

Comment:
BTW maybe also `optgroup` tag?

Reply:
I think that is working correctly in IE11, is it not?

Reply:
Not sure, I was just guessing. Here is the glitch to check: https://morning-fuel.glitch.me